docs/en/sql-reference/20-sql-functions/21-search-functions/match.md

```diff
@@ -0,0 +1,61 @@
+---
+title: MATCH
+---
+import FunctionDescription from '@site/src/components/FunctionDescription';
+
+<FunctionDescription description="Introduced or updated: v1.2.425"/>
+
+MATCH is an inverted index search function used to match the rows that meets the query conditions, note that MATCH function can only used in where clause for filter.
+
+## Syntax
+
+```sql
+MATCH( <query_fields>, <query_string> )
+```
+
+`query_fields` have tow forms. First one is specifying a single query field and the second one is specifying multiple fields containing optional boost values.
```

Comment:
tow -> two?

XXXX has two forms: One specifies ....; the other one specifies ........